build/config.props

```diff
@@ -27,6 +27,8 @@
     <NetCoreAssemblyBuildNumber Condition=" '$(NetCoreAssemblyBuildNumber)' == '' ">7</NetCoreAssemblyBuildNumber>
 
     <IsEscrowMode>false</IsEscrowMode>
+    <IsEscrowMode Condition="'$(DotNetBuildFromSource)' == 'true'">true</IsEscrowMode>
```

Comment:
How does this affect source build?

Reply:
Naming conventions for the package, I believe. Is that true, @crummel  ?

Reply:
Not really, check https://github.com/NuGet/NuGet.Client/blob/dev/build/config.props, IsEscrow is only used for VS purposes.

Reply:
I've tested source-build without IsEscrowMode and it works. I will discard this patch